jenkins-build-pack-api/Jenkinsfile

```diff
@@ -862,9 +891,11 @@ def updateServiceWithEndpointInfo(service, domain, serviceUrl, endpointKey, endp
 				g_login_token = sh (script: "curl --silent -X POST -k -v \
 					-H \"Content-Type: application/json\" \
 					 $loginUrl \
-					-d \'${payload}\' | $jq -r '.data.token'", returnStdout:true)
-
-				g_login_token = g_login_token.replaceAll("[\n\r]", "")
+					-d \'${payload}\'", returnStdout:true)
+            			def jsonParser = new groovy.json.JsonSlurper()
+				def resultJson = jsonParser.parseText(g_login_token)
```

Comment:
general comment for all the curls in this file that we are parsing. We are not checking for response status code, what if we get 4XX /5XX?

At least we have to check what data is there in resultJson.data (empty, null) before attempting replaceAll/split etc..

Reply:
suggested changes has been made

Reply:
I cant see the change..... I see following in line 909:
g_login_token = resultJson.data.token.replaceAll("[\n\r]", "")

Reply:
suggested changes has been made